includes/class-coblocks-plugin-deactivation.php

```diff
@@ -0,0 +1,128 @@
+<?php
```

Comment:
@sbalakrishnan1-godaddy I just noticed, this class file should probably go into `includes/admin/class-coblocks-plugin-deactivation.php` since it's only using WordPress admin hooks.